### DISCUSSION TITLE
CTkTextbox canot typing Unicode font please help

### DISCUSSION THREAD

**Discussion by AngRotanak** [TomSchimansky/CustomTkinter#1380] [Q&A]
![screen shot](https://user-images.githubusercontent.com/102489581/227666941-fc45f18f-ff83-4a70-a61a-418ab7fcfcb8.jpg)

**Comment by Akascape** [comment]
@AngRotanak Try changing the font of textbox.

Here is a method to import custom font files inside customtkinter.
```python
customtkinter.FontManager.load_font("my_font.ttf")  # Import the .ttf file by this method
```
After this, you can simply use it `font=(my_font,18)`.
